src/main/java/sirius/biz/tycho/kb/SynchronizeArticlesTask.java

```diff
@@ -119,7 +119,10 @@ private void updateArticle(String templatePath, String syncId) {
                                            .orElseThrow(() -> new IllegalArgumentException("Failed to load KBA: "
                                                                                            + templatePath));
             GlobalRenderContext context = tagliatelle.createRenderContext();
-            template.render(context);
+
+            if (!isRendering(template, context)) {
```

Comment:
Don't you want to check for isArticle before?

Reply:
Ah, thx... I linked the closed PR in the description, because it does not mention exactly why it was closed.